api.planx.uk/modules/pay/service/inviteToPay/createPaymentSendEvents.test.ts

```diff
@@ -3,9 +3,10 @@ import app from "../../../../server.js";
 import { createScheduledEvent } from "../../../../lib/hasura/metadata/index.js";
 import { queryMock } from "../../../../tests/graphqlQueryMock.js";
 import { flowWithInviteToPay } from "../../../../tests/mocks/inviteToPayData.js";
+import { MockedFunction } from "vitest";
```

Comment:
no `vi` namespace so we have to explicitly import types like this one, as per [docs](https://vitest.dev/guide/migration.html#types)